llvm/lib/Target/DirectX/DirectXTargetMachine.cpp

```diff
@@ -12,6 +12,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "DirectXTargetMachine.h"
+#include "DXILDataScalarization.h"
```

Comment:
Why did this need to be added? We are only using the legacy pass in this file?

Reply:
Need the pass constructor, transitively included by DirectXPassRegistry.def